bitnami/spring-cloud-dataflow/templates/skipper/deployment.yaml

```diff
@@ -80,18 +80,22 @@ spec:
               value: "7577"
             - name: SPRING_CLOUD_CONFIG_ENABLED
               value: "false"
+            - name: SPRING_CLOUD_KUBERNETES_CONFIG_ENABLE_API
+              value: "false"
             - name: SPRING_CLOUD_KUBERNETES_SECRETS_ENABLE_API
-              value: "true"
+              value: "false"
```

Comment:
Could you elaborate about why this should be false?

Reply:
Because secrets already mounted here: `/etc/secrets`. So not need to do an additional query to Kubernetes API.

It's based on Spring Cloud Data Flow documentation page (https://docs.spring.io/spring-cloud-dataflow/docs/current-SNAPSHOT/reference/htmlsingle/#configuration-kubernetes-general), quote:

**The Spring Cloud Data Flow server for Kubernetes uses the spring-cloud-kubernetes module to process secrets that are mounted under /etc/secrets. ConfigMaps must be mounted as application.yaml in the /config directory that is processed by Spring Boot. To avoid access to the Kubernetes API server the SPRING_CLOUD_KUBERNETES_CONFIG_ENABLE_API and SPRING_CLOUD_KUBERNETES_SECRETS_ENABLE_API are set to false.**